_posts/2022-01-10-hacking-netflix-eureka.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Hacking Netflix Eureka"
+excerpt: "Found explosed Eureka server and don't know what to do?"
+tags: Eureka Security 
+authors:
+- Maxim Tyukov
+header: 
+    teaser: /assets/images/post/hacking-netflix-eureka/hacking-netflix-eureka.jpeg
+    teaser_alt: Hello, Blockchain!
```

Comment:
I believe this isn't the right alt text, right?

Reply:
yes, fixed